Resources/Prototypes/Entities/Structures/Shuttles/thrusters.yml

```diff
@@ -19,6 +19,7 @@
       damage:
         types:
           Heat: 40
+          Structural: 1
```

Comment:
does this work? amazing if true

Reply:
In fact, it isn't, the code will only hit entity objects (Player / NPC etc)
I will leave it for now, as I do want to get it to work later.